docs/src/main/asciidoc/spring-native-support.adoc

```diff
@@ -0,0 +1,129 @@
+[#spring-native-support]
+== Spring Native Support
+
+=== Spring Native
+
+Spring Native provides support for compiling Spring applications to native executables using the GraalVM `native-image` compiler. We provide a native configuration extension based on `spring-aot` to support Spring Cloud Azure libraries, it will be recognized by the Spring AOT plugins and help convert them into the configuration files of the `native-image` compiler.
+
+The following are the libraries we have supported:
+
+.Supported Spring Cloud Azure Libraries
+[cols="4*", options="header"]
+|===
+|Library Artifact ID                                     |Native Image Support Version |GraalVM Compiler Version |Spring Native Version
+|spring-cloud-azure-starter-appconfiguration             |4.1.0-beta.1                 |22.0.0.2                 |0.11.3
+|spring-cloud-azure-starter-eventhubs                    |4.1.0-beta.1                 |22.0.0.2                 |0.11.3
+|spring-cloud-azure-starter-integration-eventhubs        |4.1.0-beta.1                 |22.0.0.2                 |0.11.3
+|spring-cloud-azure-starter-integration-storage-queue    |4.1.0-beta.1                 |22.0.0.2                 |0.11.3
+|spring-cloud-azure-starter-keyvault-secrets             |4.1.0-beta.1                 |22.0.0.2                 |0.11.3
+|spring-cloud-azure-starter-storage-blob                 |4.1.0-beta.1                 |22.0.0.2                 |0.11.3
+|spring-cloud-azure-starter-storage-file-share           |4.1.0-beta.1                 |22.0.0.2                 |0.11.3
+|spring-cloud-azure-starter-storage-queue                |4.1.0-beta.1                 |22.0.0.2                 |0.11.3
+|===
+
+=== Dependencies
```

Comment:
The `Dependencies` section in our other part, like Spring Data, Spring Integration, is used to show users how to add library into their application.